Analytics/resources/holt_winters.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+
+class HoltWinters:
```

Comment:
Given that the potential for use of this class is removed [here](https://github.com/FutureCitiesCatapult/SharingCitiesDashboard/pull/100/commits/2ee300e3a07206a6bb1114414c5f3424ebb4d005) should it still be included? It is valuable code but will exist in the commit history even if we remove it here.